src/contracts/signer-identity.spec.ts

```diff
@@ -0,0 +1,364 @@
+/*
+  Copyright (c) 2018-present evan GmbH.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
when do we actually use the apache license?

Reply:
Fixed license headers.